maquetado/setup/scss/titles.scss

```diff
@@ -0,0 +1,20 @@
+.title {
+  display: flex;
+  align-items: center;
+  font-size: 24px;
+  margin-top: 50px;
+  margin-left: 100px;
+
+  &::before {
+    content: '';
+    background: url(img/triangle.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    display: block; 
+    height: 90px;
+    width: 60px;  
+    position: absolute;
+    left: 55px;
```

Comment:
Always when you use `position: absolute` use two fixed position attributes, you are using left so add `top` or `bottom`